data-explorer/kusto/concepts/data-purge.md

```diff
@@ -116,8 +116,8 @@ Purge command may be invoked in two ways for differing usage scenarios:
 > To connect to a cluster using the Azure Data Explorer Web UI, see [Add clusters](../../web-query-data.md#add-clusters).
 	 
   ```kusto
-	 // Connect to the Data Management service - this command only works in Kusto.Explorer
-	 #connect "https://ingest-[YourClusterName].[region].kusto.windows.net" 
+	 // Connect to the Data Management service - this command only works in Kusto.Explorer without [database].
+	 #connect "https://ingest-[YourClusterName].[region].kusto.windows.net/[database]"
```

Comment:
/[database] is not really needed, as far as I can see - it works both in KE and in KWE without the database.
So the "- this command only works in Kusto.Explorer without [database]" part can also be deleted